handler.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import os
+from pathlib import Path
+
+import boto3
+import redis
+
+from s3_log_shipper.parsers import ParserManager
+from s3_log_shipper.s3_event_models import S3Event
+from s3_log_shipper.shipper import RedisLogShipper
+
+log: logging.Logger = logging.getLogger(__name__)
+
+
+def get_log_parser() -> ParserManager:
```

Comment:
This method seems to return a ParserManager (which itself has a method get_parser) rather than a Parser, so perhaps could rename?

Reply:
Resolved.